lib/challenge.js

```diff
@@ -1,5 +1,35 @@
 'use strict';
 
+var stringNum;
+var reversedString;
+var reversedNumber;
+var numberInBinary;
+var sum;
+
+let isPalindrome = function isPalindrome(number) {
+  stringNum = number.toString();
+  reversedString = stringNum.split('').reverse().join('');
+  reversedNumber = Number(reversedString);
+  return number === reversedNumber;
```

Comment:
This last step, of comparing `number` to `reversedNumber`, is unnecessary - you could just compare `stringNum` to `reversedString`.